driver/src/test/java/org/neo4j/driver/stress/CausalClusteringStressIT.java

```diff
@@ -30,6 +31,7 @@
 import org.neo4j.driver.exceptions.SessionExpiredException;
 import org.neo4j.driver.util.cc.LocalOrRemoteClusterExtension;
 
+@Testcontainers( disabledWithoutDocker = true )
```

Comment:
do we really want silent failure if docker isn't available?

Reply:
It would skip, not fail.
This is currently needed for our Windows builds.